use std::io;

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let stdout = io::stdout();
    if main == test {
        return Err("".into());
    }

    Ok(())
}

